tools/rna_tools/music/music.xml

```diff
@@ -0,0 +1,68 @@
+<tool id="music" name="MUSIC" version="0.1.0">
+    <description>An algorithm for identification of enriched regions at multiple scales in the read depth signals from ChIP-Seq experiments</description>
+    <requirements>
+        <requirement type="package" version="1.0.0">music</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+        cat "$input1" | MUSIC -preprocess SAM stdin .
```

Comment:
One of them is probably a better choice: https://github.com/galaxyproject/tools-iuc/